apps/website-25/src/components/careers/CareersCallout.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { CTAButton } from '@bluedot/ui';
+
+const Callout: React.FC = () => {
+  const title = 'Join us in our mission to ensure humanity safely navigates the transition to transformative AI.';
+  const ctaLabel = 'View our careers';
+  const ctaLink = '/careers';
+
+  return (
+    <div className="callout bg-[radial-gradient(ellipse_at_30%,_#fff_0%,_#6687ff_100%)] rounded-lg p-8 m-8 flex flex-col items-center">
+      <h2 className="callout__title text-black font-bold text-2xl mb-4 text-center">{title}</h2>
```

Comment:
do you need these extra styles? we should have default h2 styling thanks to the set-up in [globals.css](https://github.com/bluedotimpact/bluedot/blob/a93b29b23ee9ba172b405f9fd04650aa629a71df/apps/website-25/src/globals.css#L8)
```suggestion
      <h2 className="callout__title mb-4 text-center">{title}</h2>
```